ovs/dal/hybrids/storagerouter.py

```diff
@@ -145,3 +148,31 @@ def _recovery_domains(self):
         :return: List of domain guids
         """
         return [junction.domain_guid for junction in self.domains if junction.backup is True]
+
+    def _features(self):
+        """
+        Returns information about installed/available features
+        :return: Dictionary containing edition and available features per component
+        """
+        client = SSHClient(self, username='root')
```

Comment:
Retrieving offline StorageRouters or StorageRouters on which the heartbeat data is old, will suffer from this line